docs/contributors/localizing.md

```diff
@@ -1,6 +1,8 @@
 # Localizing Gutenberg Plugin
 
-To translate Gutenberg in your locale or language, [select your locale here](https://translate.wordpress.org/projects/wp-plugins/gutenberg) and translate _Development_ (which contains the plugin's string) and/or _Development Readme_ (please translate what you see in the Details tab of the [plugin page](https://wordpress.org/plugins/gutenberg/)).
+As general WordPress Plugins, Gutenberg Plugin is translated via translation management system at https://translate.wordpress.org. If you are not familiar with this system and translation process, refer [this document](https://make.wordpress.org/polyglots/handbook/tools/glotpress-translate-wordpress-org/) first.
```

Comment:
I'm not a native English speaker, but I've got the impression there are some articles & prepositions missing here. A native English speaker should review this before we merge it...